ofl/russolo/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,17 @@
+<p>
+Russolo is a typeface combining Bodoni and Futura into a high-contrast geometric sans.
+
+When I was a student at RIT in 2008-2010, I studied typography under the mentorship of Chuck Bigelow. I worked on a research project intended to find accurate revivals by comparing high-resolution scans of original prints to digitizations. Then it expanded to quantifying what the “average” typeface looked like from the main eras of typographic development by measuring various typefaces from those eras. Finally, I thought that it would be interesting to use my data to combine unique designs into something new (or at least interesting).
```

Comment:
@laerm0 a double CR doesn't start a new paragraph in html. If you'd like each text block to be a new paragraph, do the following:

```
<p>
paragraph1
</p>
<p>
paragraph2
</p>
...
```

Reply:
Basic HTML! A mystery. 😂